core/iwasm/common/wasm_memory.c

```diff
@@ -685,19 +699,29 @@ wasm_enlarge_memory_internal(WASMModuleInstance *module, uint32 inc_page_count)
     os_writegsbase(memory_data_new);
 #endif
 
+return_func:
+    if (!ret && enlarge_memory_error_cb)
+        enlarge_memory_error_cb(inc_page_count, total_size_old, 0,
+                                failure_reason, (wasm_module_inst_t)module,
+                                module->e->common.exec_env);
```

Comment:
Should distinguish module->e, it may be WASMModuleInstanceExtra or AOTModuleInstanceExtra